drivers/mbox/mbox_andes_plic_sw.c

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2022 Andes Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/mbox.h>
+
+#define LOG_LEVEL CONFIG_MBOX_LOG_LEVEL
+#include <zephyr/logging/log.h>
+#include <zephyr/irq.h>
+LOG_MODULE_REGISTER(mbox_andes_plic_sw);
+
+#define DT_DRV_COMPAT	andestech_plic_sw
+
+/* register definitions */
+#define PRIORITY_BASE 0x0
+#define PENDING_BASE  0x1000
+#define ENABLE_BASE   0x2000
+#define CLAIM_BASE    0x200004
+
+#define IRQ_REG(n) (n >> 5)
```

Comment:
can you indent these?

Reply:
OK, I got it.
We have update the source code.